misc/jwt.go

```diff
@@ -0,0 +1,89 @@
+package misc
+
+import (
+	"fmt"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/labbsr0x/goh/gohtypes"
+	"github.com/spf13/viper"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+// GenerateToken generates a jwt token
+func GenerateToken(data jwt.MapClaims) (string, error) {
+	secret := viper.GetString("secret-key")
```

Comment:
@claudiosegala, we should avoid resolving any config outside the config package.